src/renderers/webgl/WebGLUniforms.js

```diff
@@ -597,6 +613,50 @@ function setValueT1Array( gl, v, textures ) {
 
 }
 
+function setValueT2DArray1Array( gl, v, textures ) {
```

Comment:
Why have you included `setValueT2DArray1Array()` and `setValueT3D1Array()` into this PR? It seems to be something different than `Add `internalFormat` to Texture`.

Reply:
You are right. Sometimes you know you think about something and you get too crazy about it. Removed and I will do another PR for all of that.